CONTRIBUTING.md

```diff
@@ -92,6 +92,31 @@ the image owner to implement it.
 1. All images must provide a `CMD` or `ENTRYPOINT`. If your image is designed
 to be extended, then this should output documentation on how to extend the
 image to be useful.
+1. Use `LABEL` instructions for additional information such as ports and volumes. The following are common label instructions that should be present in all images where applicable:
+
+| Label   | Value | Applicability |
+| -------- | ----- | ------------- |
+| provider | The provider of the image (should be `Oracle` for base images in this repo) | All images |
+| issues | URL where issues and questions can be posted (should be `https://github.com/oracle/docker-images/issues` for base images in this repo | All images |
+| maintainer | Name of the maintainer team | All images |
+| volume(.`purpose`) | Use `volume` labels to describe the volumes of an image.<br/>If your image has multiple volumes, use qualified names to specify the purpose of each volume, for example `volume.data` for data to be persisted outside the container.<br/>Use hierarchical nesting for multiple volumes of the same type, for example:<br/><ul><li>`volume.data.dir1`</li><li>`volume.data.dir2`</li></ul> | For all images that have data that should be persisted outside the container |
+| port(.`purpose`) | Use `port` labels to describe the ports of an image.<br/>If your images has multiple ports, use qualified names to specify the purpose of each port, for example `port.app` for the port on which your application is reachable.<br/>Use hierarchical nesting for multiple ports of the same type, for example:<br/><ul><li>`port.app.http`</li><li>`port.app.https`</li></ul> | For all images that have ports that should be exposed outside the container |
+
+For example, for the Oracle Database 18c XE image we use the following labels:
+
+```
+LABEL "provider"="Oracle Corporation"                                   \
```

Comment:
The example needs to be updated to just `Oracle` per our discussion.

Reply:
Fixed